openwrt-relayd-cve-2020-11752/README.md

```diff
@@ -0,0 +1,47 @@
+# OpenWRT mdns CVE-2020-11752
```

Comment:
Should header say "relayd"? Also want to check that the CVE number isn't a left-over from copy-paste.